File: m2cgen/interpreters/javascript/linear_algebra.js
function addVectors(v1, v2) {
    let result = new Array(v1.length);

    for (let i = 0; i < v1.length; i++) {
        result[i] = v1[i] + v2[i];
    }

    return result;
}

function mulVectorNumber(v1, num) {
    let result = new Array(v1.length);

    for (let i = 0; i < v1.length; i++) {
        result[i] = v1[i] * num;
    }

    return result;
}

File: m2cgen/interpreters/go/linear_algebra.go
func addVectors(v1, v2 []float64) []float64 {
    result := make([]float64, len(v1))
    for i := 0; i < len(v1); i++ {
        result[i] = v1[i] + v2[i]
    }
    return result
}
func mulVectorNumber(v1 []float64, num float64) []float64 {
    result := make([]float64, len(v1))
    for i := 0; i < len(v1); i++ {
        result[i] = v1[i] * num
    }
    return result
}